proto/pbcommon/common.proto

```diff
@@ -150,6 +150,14 @@ message QueryMeta  {
    // Having `discovery_max_stale` on the agent can affect whether
    // the request was served by a leader.
    string ConsistencyLevel = 4;
+
+   // Reserved for NotModified and Backend.
+   reserved 5, 6;
```

Comment:
just out of curiosity -- how did you come across this?

Reply:
I spotted that these fields were present [in `structs.go`](https://github.com/hashicorp/consul/blob/db5ee0e4d21c039a990889fcc7d59ec6bf7d4673/agent/structs/structs.go#L380-L387) but not here (when adding `ResultsFilteredByACLs`) so I decided to reserved their spaces so that we don't end up with inconsistent ordering if we add them to the protobuf message in the future.